docs-gen/project/LagomIbmExamplesPlugin.scala

```diff
@@ -0,0 +1,56 @@
+import sbt._
+import Keys._
+import com.lightbend.paradox.sbt.ParadoxPlugin
+
+import scala.collection.immutable
+
+object LagomIbmExamplesPlugin extends sbt.AutoPlugin {
+  override def requires = ParadoxPlugin
+
+  override def trigger = allRequirements
+
+  object autoImport {
+    val baseUrl = settingKey[String]("")
+    val baseProject = settingKey[String]("")
+    val templateName = settingKey[String]("")
+    val bodyPrefix = settingKey[String]("")
+    val bodyTransformation = settingKey[String => String]("")
+  }
+
+  import autoImport._
+  import ParadoxPlugin.autoImport._
+
+  override def projectSettings: Seq[Setting[_]] = Seq(
+    baseUrl := "https://github.com/lagom/ibm-integration-examples/blob/master",
+    crossPaths := false,
+    // Copy README.md file
+    sourceDirectory in(Compile, paradox) := {
+      val outDir = (managedSourceDirectories in Compile).value.head / "paradox"
+      val outFile = outDir / "index.md"
+      val inDir = baseDirectory.value / ".." / ".." / baseProject.value
+      val inFile = inDir / "README.md"
+      IO.write(outFile, bodyPrefix.value + bodyTransformation.value(IO.read(inFile)))
+      if ((inDir / "tutorial").exists) {
+        IO.copyDirectory(inDir / "tutorial", outDir / "tutorial")
+      }
+      outDir
+    },
+    paradoxProperties += ("download_url" -> s"https://example.lightbend.com/v1/download/${templateName.value}"),
+    bodyPrefix :=
```

Comment:
Unlike Akka samples you guys already put titles in the README, so now it's duplicated.

![lagom_integration_with_ibm_cloud_object_storage](https://user-images.githubusercontent.com/184683/31030127-3a9b7202-a522-11e7-90f3-15ed55c9957f.png)

Reply:
:squinting: Hmmm, so I can remove `.copy(description = title)` ? (I've never used paradox I'm affraid)

Reply:
I think in your case you can keep the `bodyPrefix to := ""`.